README.md

```diff
@@ -53,6 +53,8 @@ indirect business models around the free open-source software platform
 provided by RIOT, e.g. it is possible to link closed-source code with the
 LGPL code.
 
+[Ariel OS](https://github.com/ariel-os/ariel-os) is an offspring of RIOT written in Rust following RIOT's targets and vision.
```

Comment:
> s/targets/vision? I mean, we even have a doc page for that: https://doc.riot-os.org/vision.html

Maybe this page also should be linked then ;-)

Reply:
```suggestion
[Ariel OS](https://github.com/ariel-os/ariel-os) is an offspring of RIOT written in Rust following RIOT's targets and [vision](https://doc.riot-os.org/vision.html).
```